dev_guide/builds/build_inputs.adoc

```diff
@@ -854,8 +854,8 @@ secrets are placed in the working directory of the builder image.
 
 The same rule is used when a `destinationDir` is a relative path; the secrets
 are placed in the paths that are relative to the image's working directory. The
-`destinationDir` must exist or an error will occur. No directory paths are
-created during the copy process.
+parent directory of `destinationDir` must exist or an error will occur. 
+Only the `destinationDir` is created during the copy process.
```

Comment:
"parent directory of destinationDir" doesn't sound quite right.

A few scenarios we need to make sure we understand:

1) /etc/nonexistent1/nonexisting2

2) /etc/nonexistent

3) /etc/existing

4) ./existing/nonexisting

5) ./nonexisting1/nonexisting2

6) ./existing/nonexisting

7) ./existing

I suspect what really needs to be said is "only the final path segment of "destinationDir" will be created, the parent tree must exist".  And then we need to confirm what happens if the entire tree(including the final segment) already exists.

Reply:
absolute paths:
1. -     /etc/nonexistent1/nonexisting2    -- build fail, only creates nonexistent1 dir.
2. -     /etc/nonexistent                            -- will create /etc/nonexistent dir.
3. -     /etc/existing                                  --  only do copying operation.
relative paths:
1.     ./existing/nonexisting1/nonexistin2    -- build fail, only creates nonexistent1 dir.
2.     ./nonexisting                                      --  ./nonexisting created
3.     ./existing                                            -- only copy
If the entire tree(including the final segment) already exists, just copy cm/secret to destinationdir.